lib/Parse/Lexer.cpp

```diff
@@ -2386,6 +2386,7 @@ void Lexer::lexImpl() {
   case 0:
     switch (getNulCharacterKind(CurPtr - 1)) {
     case NulCharacterKind::CodeCompletion:
+      while (advanceIfValidContinuationOfIdentifier(CurPtr, BufferEnd));
```

Comment:
Nitpick: can you put the semicolon on its own line to make it more obvious? Or use `{}` instead?